ui/app/templates/components/secret-edit.hbs

```diff
@@ -53,43 +53,100 @@
     <label for="json" class="has-text-grey">JSON</label>
   </div>
   {{#if (and (eq mode 'show') (or canEditV2Secret canEdit))}}
-    <div class="control">
       {{#let (concat 'vault.cluster.secrets.backend.' (if (eq mode 'show') 'edit' 'show')) as |targetRoute|}}
-        {{#if isV2}}
-          <LinkTo
-            @params={{array targetRoute model.id (query-params version=this.modelForData.version)}}
-            @replace={{true}}
-            class="link link-plain has-text-weight-semibold"
-          >
-            Create new version
-          </LinkTo>
-        {{else}}
-          <LinkTo
-            @params={{array targetRoute model.id}}
-            @replace={{true}}
-            class="link link-plain has-text-weight-semibold"
-          >
-            Edit Secret
-          </LinkTo>
-        {{/if}}
-      {{/let}}
-    </div>
+        {{#unless (and isV2 (or modelForData.destroyed modelForData.deleted))}}
+          <div class="control">
+            <BasicDropdown
+              @class="popup-menu"
+              @horizontalPosition="auto-right"
+              @verticalPosition="below"
+              @onClose={{action "hideWrappedData"}}
```

Comment:
Aha I see how it's working now. What do you think about this clearing out `wrappedData` and then you could have `showWrapButton: not('wrappedData')` - not a big deal but you wouldn't have to set `showWrapButton` in 3 places then, it'd update automatically.